tc/core/polyhedral/cuda/mapped_scop.cc

```diff
@@ -357,7 +357,7 @@ size_t MappedScop::mapToThreads(detail::ScheduleTree* band) {
 
   // Immediately return if mapping to one thread dimension only was requested
   // and a reduction was already mapped.  (Note that reduction is detected only
-  // if there are not enough outer coincident members, 0 in this case).
+  // if there are enough outer coincident members, 0 in this case).
```

Comment:
I'm not sure about this one. AFAIU, reduction it _not_ detected if there is 1 coincident member followed by 1 reduction member, and mapping to only 1 thread dimension is required.  It _is_ detected if mapping to 2 thread dimensions is required.  That's what I interpreted as having _not enough_ coincident members.  Maybe it's better to reformulate this as "reduction is detected only if there are less coincident dimensions than thread dimensions requested for mapping" to avoid confusion.